OpenEdXMobile/src/main/java/org/edx/mobile/view/AuthenticatedWebViewFragment.kt

```diff
@@ -92,7 +97,19 @@ open class AuthenticatedWebViewFragment : BaseFragment() {
                     false,
                     isManuallyReloadable,
                     false,
-                    null
+                    null,
+                    object: PageFinishCallback{
+                        override fun onUrlOverriderClick(onlyDismiss : Boolean) {
+                            if (!onlyDismiss) {
+                                environment.router.showAuthenticatedWebViewActivity(
```

Comment:
we to define some action, cuz app redirect to delete my account page on other URL clicks(newly added).